src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -338,12 +331,30 @@ export abstract class IoTWorkbenchProjectBase {
       projectConfig[`${ConfigKey.workbenchVersion}`] = workbenchVersion;
 
       await FileUtility.writeJsonFile(
-          type, iotworkbenchprojectFilePath, projectConfig);
+          type, this.iotWorkbenchProjectFilePath, projectConfig);
     } catch (error) {
-      throw new Error(`Generate or update ${
-          FileNames.iotworkbenchprojectFileName} file failed: ${
-          error.message}`);
+      throw new Error(
+          `Update ${FileNames.iotworkbenchprojectFileName} file failed: ${
+              error.message}`);
+    }
+  }
+
+  /**
+   * Get project configs from iot workbench project file
+   * @param type Scaffold type
+   */
+  async getProjectConfig(type: ScaffoldType): Promise<{[key: string]: string}> {

```

Comment:
Since the function return JSON object, can we just return any instead of{[key:string]:string} ?

Reply:
Fixed.